docs/source/yaml/world_schema.rst

```diff
@@ -29,6 +29,7 @@ The world schema looks as follows, where ``<angle brackets>`` are placeholders:
            y: <y>
          rotation_eul:
            yaw: <yaw>
+         angle_units: <units> # Units can be Radians or Degrees
```

Comment:
style: should use double space between code and inline comment.

Also should be more specific about the actual valid strings

```suggestion
         angle_units: <units>  # Can be "radians" (default) or "degrees"

Reply:
1. I am a bit confused about how to specify the color. Currently, it seems we can specify the color in three different ways. Should we include all options like this?

  ```
  color:  [<r>, <g>, <b>] or <"color_name"> or <"hexadecimalcode">
  ```


2. Additionally, I have a doubt regarding `nav_poses`. Is the following format correct?

  ```
  nav_poses:
           - [<x1>, <y1>, <z1>, <yaw1> <units>]
  ```
3. I observed in `location_schema.rst` and `object_schema.rst` that examples are provided after specifying the schema. Should I also update the examples to match the changed schema (for example color format and angle_units in nav_poses ?

Reply:
1. That looks great as is!

2. No, I also didn't do a good job of updating them. This should be something like:

```yaml
nav_poses:
  - position:
      x: <x1>
      y: <y1>
  - rotation_eul:
      yaw: <yaw1>
      angle_units: degrees
  - position:
      x: <x2>
      y: <y2>
    rotation_quat:
      w: 0.707
      z: 0.707
```

... which actually brings up a good point. Would it make sense to have the `angle_units`  be inside the `rotation_eul` key instead of at the top of the dict?

3. I'm fine either way, can do this later. If you want I can pass through and clean up these docs after this PR goes in...

Reply:
2. I agree with your suggestion, it makes sense to include units in the rotation when using Euler angles. I have also updated the schema of `nav_poses` as per your feedback...let me know if I missed anything.
3. I have made the changes. Could you please review them? If any further tweaks are needed, I will do